distro/pom.xml

```diff
@@ -15,46 +15,46 @@
         <episodesVersion>1.1.0</episodesVersion>
         <eventModuleVersion>2.10.0</eventModuleVersion>
         <owaVersion>1.13.0</owaVersion>
-        <htmlwidgetsVersion>1.10.0</htmlwidgetsVersion>
+        <htmlwidgetsVersion>1.11.0</htmlwidgetsVersion>
         <idgenModuleVersion>4.7.0</idgenModuleVersion>
         <idgenWebServicesModuleVersion>1.4.0</idgenWebServicesModuleVersion>
         <legacyUiOmodVersion>1.13.0</legacyUiOmodVersion>
-        <mailappenderVersion>0.94.3</mailappenderVersion>
-        <metadataMappingVersion>1.5.0</metadataMappingVersion>
-        <metadataSharingVersion>1.8.0</metadataSharingVersion>
-        <openMRSVersion>2.5.10</openMRSVersion>
+        <mailappenderVersion>0.95.0-SNAPSHOT</mailappenderVersion>
+        <metadataMappingVersion>1.6.0</metadataMappingVersion>
+        <metadataSharingVersion>1.9.0</metadataSharingVersion>
+        <openMRSVersion>2.5.12</openMRSVersion>
         <operationTheaterVersion>1.7.0</operationTheaterVersion>
         <providerManagementVersion>2.13.0</providerManagementVersion>
         <rulesEngineVersion>1.0.0</rulesEngineVersion>
         <reportingVersion>1.23.0</reportingVersion>
-        <serializationVersion>0.2.15</serializationVersion>
+        <serializationVersion>0.2.16</serializationVersion>
         <uicommonsModuleVersion>2.19.0</uicommonsModuleVersion>
         <uiframeworkModuleVersion>3.22.0</uiframeworkModuleVersion>
         <uilibraryVersion>2.0.7</uilibraryVersion>
-        <webServicesRestVersion>2.39.0</webServicesRestVersion>
-        <reportingRestVersion>1.12.0</reportingRestVersion>
-        <bahmniIEOmodVersion>1.3.0</bahmniIEOmodVersion>
-        <appointmentsVersion>1.7.0</appointmentsVersion>
-        <pacsQueryVersion>1.4.0</pacsQueryVersion>
-        <fhir2ModuleVersion>1.8.0</fhir2ModuleVersion>
-        <fhir2ExtensionModuleVersion>1.2.0</fhir2ExtensionModuleVersion>
-        <openConceptLabVersion>1.3.0</openConceptLabVersion>
-        <initializerModuleVersion>2.4.0</initializerModuleVersion>
-        <bahmniCommonsVersion>1.0.0</bahmniCommonsVersion>
-        <teleconsultationVersion>1.1.0</teleconsultationVersion>
-        <communicationVersion>1.1.0</communicationVersion>
+        <webServicesRestVersion>2.40.0</webServicesRestVersion>
+        <reportingRestVersion>1.14.0</reportingRestVersion>
+        <bahmniIEOmodVersion>1.4.0-SNAPSHOT</bahmniIEOmodVersion>
+        <appointmentsVersion>1.8.0-SNAPSHOT</appointmentsVersion>
+        <pacsQueryVersion>1.5.0-SNAPSHOT</pacsQueryVersion>
+        <fhir2ModuleVersion>1.10.0-SNAPSHOT</fhir2ModuleVersion>
+        <fhir2ExtensionModuleVersion>1.3.0-SNAPSHOT</fhir2ExtensionModuleVersion>
+        <openConceptLabVersion>2.1.0</openConceptLabVersion>
+        <initializerModuleVersion>2.5.2</initializerModuleVersion>
+        <bahmniCommonsVersion>1.1.0-SNAPSHOT</bahmniCommonsVersion>
+        <teleconsultationVersion>1.2.0-SNAPSHOT</teleconsultationVersion>
```

Comment:
same question here. Are there feature from teleconsultation 1.2.0-SNAPSHOT that we are using?

Reply:
reverted this change.